node.gypi

```diff
@@ -188,7 +188,15 @@
             },
           },
           'conditions': [
-            ['OS!="aix" and node_shared=="false"', {
+            ['OS!="aix" and node_shared=="false" and node_use_large_pages=="true"', {
+              'ldflags': [
+                 '-Wl,-T <(PRODUCT_DIR)/../../ld.script',
```

Comment:
Have you tested this on on other platforms than x86_64 Linux?

Reply:
No only on x86_64 Linux, specifically Ubuntu 16.04 is what I have done most of my measurement and testing on.

Reply:
Thanks fixed it.